www-client/seamonkey/seamonkey-2.53.10.2.ebuild

```diff
@@ -0,0 +1,557 @@
+			build/unix/run-mozilla.sh || die
+	fi
+
+	# Ensure that are plugins dir is enabled as default
+	sed -i -e "s:/usr/$(get_libdir)/mozilla/plugins:/usr/$(get_libdir)/${PN}/plugins:" \
+		xpcom/io/nsAppFileLocationProvider.cpp || die
+
+	# Don't exit with error when some libs are missing which we have in
+	# system.
+	sed '/^MOZ_PKG_FATAL_WARNINGS/s@= 1@= 0@' \
+		-i comm/suite/installer/Makefile.in || die
+	# Don't error out when there's no files to be removed:
+	sed 's@\(xargs rm\)$@\1 -f@' \
+		-i toolkit/mozapps/installer/packager.mk || die
+
+	# Don't build libs-% locale files for chatzilla if we are not building chatzilla
+	# (this is hard-coded in the build system at present rather than being based on configuration)
+	if ! use chatzilla ; then
+		sed '/extensions\/irc\/locales libs-/s@^@#@' \
+			-i comm/suite/locales/Makefile.in || die
+	fi
+
+	eautoreconf old-configure.in
+	cd js/src || die
+	eautoconf old-configure.in
+}
+
+src_configure() {
+	MEXTENSIONS="default"
+	# Google API keys (see http://www.chromium.org/developers/how-tos/api-keys)
+	# Note: These are for Gentoo Linux use ONLY. For your own distribution, please
+	# get your own set of keys.
+	_google_api_key=AIzaSyDEAOvatFo0eTgsV_ZlEzx0ObmepsMzfAc
+
+	######################################
+	#
+	# mozconfig, CFLAGS and CXXFLAGS setup
+	#
+	######################################
+
+	mozconfig_init
+
+	##################################
+	# Former mozconfig_config() part #
+	##################################
+
+	# Migrated from mozcoreconf-2
+	mozconfig_annotate 'system_libs' --with-system-bz2
+	mozconfig_annotate 'system_libs' --with-system-zlib
+
+	# Disable for testing purposes only
+	mozconfig_annotate 'Upstream bug 1341234' --disable-stylo
+
+	# Must pass release in order to properly select linker via gold useflag
+	mozconfig_annotate 'Enable by Gentoo' --enable-release
+
+	# Must pass --enable-gold if using ld.gold
+	if tc-ld-is-gold ; then
+		mozconfig_annotate 'tc-ld-is-gold=true' --enable-gold
+	else
+		mozconfig_annotate 'tc-ld-is-gold=false' --disable-gold
+	fi
+
+	# Enable position independent executables
+	mozconfig_annotate 'enabled by Gentoo' --enable-pie
+
+	mozconfig_use_enable debug
+	mozconfig_use_enable debug tests
+	if ! use debug ; then
+		mozconfig_annotate 'disabled by Gentoo' --disable-debug-symbols
+	else
+		mozconfig_annotate 'enabled by Gentoo' --enable-debug-symbols
+	fi
+
+	mozconfig_use_enable startup-notification
+
+	# wifi pulls in dbus so manage both here
+	mozconfig_use_enable wifi necko-wifi
+	if use kernel_linux && use wifi && ! use dbus ; then
+		echo "Enabling dbus support due to wifi request"
+		mozconfig_annotate 'dbus required by necko-wifi on linux' --enable-dbus
+	else
+		mozconfig_use_enable dbus
+		mozconfig_annotate 'disabled' --disable-necko-wifi
+	fi
+
+	# These are enabled by default in all mozilla applications
+	mozconfig_annotate '' --with-system-nspr --with-nspr-prefix="${SYSROOT}${EPREFIX}"/usr
```

Comment:
Nit: for EAPI 7+, ${SYSROOT}${EPREFIX} is ${ESYSROOT}, so let's replace every instance of it with ${ESYSROOT}.